src/middleware/mod.rs

```diff
@@ -55,6 +55,7 @@ use std::sync::Arc;
 pub use http_client::{Body, HttpClient, Request, Response};
 
 pub mod logger;
+pub mod redirect;
```

Comment:
In Tide we only expose `tide::Redirect` -- I think it would make sense for us to do the same here.

Reply:
See also https://github.com/http-rs/surf/pull/198